Assets/Scripts/Save System/DataSaveManager.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Cosmobot
+{
+    public class DataSaveManager : SingletonSystem<DataSaveManager>
+    {
+        private GameData game_data;
+
+        [SerializeField]
+        private const string default_save_file_name = "save_file";
```

Comment:
`const` don't need to be `[SerializeField]`

Reply:
fixed